src/lib.rs

```diff
@@ -40,17 +40,19 @@ use std::time::{Duration, Instant};
 
 use reqwest::{header, RequestBuilder, Url};
 use serde::de::DeserializeOwned;
-use serde::Deserialize;
+use serde::{Deserialize, Serialize};
 use tokio::sync::{Mutex, MutexGuard};
 use tokio_compat_02::FutureExt;
 
 pub use authorization_url::*;
 pub use endpoints::*;
+use futures_util::TryFutureExt;
 /// Re-export from [isocountry](https://crates.io/crates/isocountry).
 pub use isocountry::CountryCode;
 /// Re-export from [isolanguage-1](https://crates.io/crates/isolanguage-1).
 pub use isolanguage_1::LanguageCode;
 pub use model::*;
+use std::borrow::Borrow;
```

Comment:
Move this to the group of `std` imports above this, I think it makes it cleaner.